README.md

```diff
@@ -1,6 +1,14 @@
-# Try next with this example:
-## [Run locally in prod-like environment](#run-locally-in-prod-like-environment) <br>
-## [Deploy and run on Kalix Platform on Cloud Provider of your choice ](deploy-and-run-on-kalix-platform-on-cloud-provider-of-your-choice)
+# Prerequisite
+Java 17 or later<br>
+Maven 3.6 or later<br>
+Docker 20.10.14 or higher (to run locally)<br>
+# Introduction
+
+If you are continuing the journey from [Explore Kalix Virtually]() and want the use case locally and/or deploy to Kalix Platform continue with these topics:
+- [Run locally in prod-like environment](#run-locally-in-prod-like-environment) <br>
+- [Deploy and run on Kalix Platform on Cloud Provider of your choice ](deploy-and-run-on-kalix-platform-on-cloud-provider-of-your-choice)
+
+If you want to evaluate the Kalix developer experience locally as done in the [Explore Kalix Virtually]() we suggest you follow the instructions from the beginning and to build your own project.    
 # Kalix Trial - eCommerce - Java
 ## Designing Kalix Services
 ### Use case 
```

Comment:
I think this flow  goes better :


If you have gone through the Explore Kalix Journey, you are all set to :

1. Run this project in your local machine(in a dockerized Kalix Runtime).
2. Deploy and Run in on the Kalix Cloud Platform.

However, If you’d like better understanding the Kalix Developer Experience better and how to build applications on Kalix, here’s more details on the same-use.